docs: add dotnet installation & example doc (#1567)
microsoft/SynapseML: website/docs/getting_started/installation.md

To try out SynapseML using the R autogenerated wrappers [see our
instructions](reference/R-setup.md).  Note: This feature is still under development
and some necessary custom wrappers may be missing.

## C# (.NET)

> To try out SynapseML with .NET, please follow [instructions](reference/dotnet-setup.md).
> 
> Note: This feature is not fully supported so some special functions may be missing.

Reviewer: ```suggestion
To try out SynapseML with .NET, please follow the [.NET Installation Guide](website/docs/reference/dotnet-setup.md).
```
